client/driver/fabfile.py

```diff
@@ -180,60 +193,61 @@ def change_conf(next_conf=None):
     assert isinstance(recommendation, dict)
 
     for name, value in recommendation.items():
-        if CONF['database_type'] == 'oracle' and isinstance(value, str):
+        if dconf.DB_TYPE == 'oracle' and isinstance(value, str):
             value = value.strip('B')
         lines.append('{} = {}\n'.format(name, value))
     lines.append('\n')
 
-    tmpconf = 'tmp_' + os.path.basename(CONF['database_conf'])
-    with open(tmpconf, 'w') as f:
+    tmp_conf_out = os.path.join(dconf.TEMP_DIR, os.path.basename(dconf.DB_CONF) + '.out')
+    with open(tmp_conf_out, 'w') as f:
         f.write(''.join(lines))
 
-    local('sudo cp {0} {0}.ottertune.bak'.format(CONF['database_conf']))
-    local('sudo mv {} {}'.format(tmpconf, CONF['database_conf']))
+    run('cp {0} {0}.ottertune.bak'.format(dconf.DB_CONF))
```

Comment:
On my local machine, run this command causes permission deny.
sudo() works fine.